### ISSUE TITLE
solar circuit not detected

### ISSUE THREAD

**Issue by JackMotion4GitHub** [bosch-thermostat/home-assistant-bosch-custom-component#41]
Hi.

I have been using this bosch integration with Home Assistant for several months and it kept presenting a lot of useful data about my Buderus heatpump with its heating circuit hc1, water heating circuit dhw1 and solar circuit sc1. The heatpump is connected to a KM200 interface.

Unfortunately after upgrading to v0.9.3, the solar circuit sc1 is not detected anymore. Same with 0.9.4 and latest 0.9.5 versions.

During initial configuration (using IVT with XMPP for the RC300 series device), there are only 4 areas detected instead of 5 areas previously.

I am running the latest Home Assistant build (currently 0.118.5).  I have tried to completely remove the integration configuration and the component, rebooted Home Assistant, then re-added the component and re-done the config, but the result was the same.

Here is my bosch_scan.json: https://pastebin.com/jsBXARGE

Thanks for any help.

**Comment by pszafer** [comment]
There was a bug introducing configuration for `Bosch Compress 7001i`
This device has gateway capability as well as RC300 in your setup.
so for this library you have 2 gateways.
as from new version I added support for solar circuit with only heatpump,
but as well I added that KM200 is first choice if multiple devices are detected.

try new version: https://github.com/bosch-thermostat/home-assistant-bosch-custom-component/releases/tag/v0.9.7
